tensorflow/core/util/BUILD

```diff
@@ -405,7 +405,6 @@ cc_library(
         "mkl_util.h",
     ],
     copts = tf_copts() + ["-fexceptions"],
-    linkstatic = 1,
```

Comment:
I was also wondering whether this is needed. Glad it isn't. Thank you! :)

Reply:
@penpornk Looks like ready to pull was removed. Is there anything needed from our side?